### ISSUE TITLE
pip install beeware failed on Ubuntu 16.04

### ISSUE THREAD

**Issue by arlejeun** [beeware/beeware#5]
I'm trying to install beeware on my virtual environment venv - Linux Ubuntu 16.04 but it failed. It worked like a charm on MacOS though. Below are the steps I followed

`python3 -m venv venv`
`. venv/bin/activate`
`pip install beeware`

During the installation, I get this error in my terminal

`ImportError: /home/alejeune/Development/beeware/pycon/venv/lib/python3.6/site-packages/cairo/_cairo.cpython-36m-x86_64-linux-gnu.so: undefined symbol: cairo_tee_surface_index`

See below the command highlited in my terminal

`Command "/home/alejeune/Development/beeware/pycon/venv/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-yqr0u95l/pygobject/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-3kdlzyar/install-record.txt --single-version-externally-managed --compile --install-headers /home/alejeune/Development/beeware/pycon/venv/include/site/python3.6/pygobject" failed with error code 1 in /tmp/pip-install-yqr0u95l/pygobject/`

I'm new with Cairo but it looks like they migrated to pyCairo. I've also seen this issue with pygobject.
https://github.com/pygobject/pycairo/issues/105

Can you please suggest how to address this issue?

**Comment by Huyston** [comment]
I can't install either because cairo fails to install. Skipping cairo makes possible to create the project tree using beeware, but the app fails to launch.

**Comment by apoorvpatne10** [comment]
Same issue here. Here's the entire [traceback](https://pastebin.com/3usZUsGJ). Using python 3.6 on ubuntu 18.04.

Tried 
```
sudo apt-get install python3-cairo
sudo apt-get install python-cairo
```
But did no good.

**Comment by ocobacho** [comment]
I had the same issue. I went to pycairo docs https://pycairo.readthedocs.io/en/latest/getting_started.html In the documentation the show the following dependencies for ubuntu.
` sudo apt install libcairo2-dev pkg-config python3-dev. `
After installing them I try to install beeware with pip. 
`pip install beeware`
It show me another error where it sugested to install  
`sudo apt install libgirepository1.0-dev`
After installing that the installation of beeware didn't show any more errors

**Comment by maramaraz** [comment]
I had similar issues. One with headers (libraries) another one related to my python being Anaconda distro. I managed to solve both of them in a following way:
1. Headers and dependencies
I was unable to install ```pycairo``` because I couldn't install ```cairo``` headers with apt-get. Problem was that ```cairo``` headers were dependant on headers that were dependant on ```zlib1g``` older than the one I had installed on my system. So I had to downgrade ```zlib1g``` and following line made the trick:
```sudo apt-get install zlib1g-dev=1:1.2.8.dfsg-2ubuntu4```
By adding ```=``` after package name I specified the version I want to reinstall and it worked. Following this one I normally installed ```cairo``` headers by running:
```sudo apt-get install libcairo2-dev```

2. Issue related to Anaconda
was that I couldn't use "normal" virtual environments without issues. So idea was to make conda virtualenv instead with this:
```conda create --name BeeWareDev python=[version here] --channel conda-forge```
Where ```[version here]``` is the version of Python you want to run your BeeWare on. 3.6.8 in my case.
Next step is to make sure that you are running your BeeWare within newly forged dev environment. which is done by running following:
```conda activate dev``` (if it doesn't work try ```conda init bash``` -> restart the Terminal -> if you see ```(base)>``` then it's all good and try to rerun ```conda activate dev```)

Next thing is to install all of the needed packages within this environment as stated in [BeeWare Getting Started Tutorial](https://briefcase.readthedocs.io/en/latest/tutorial/getting-started.html) and you are ready to go.
